File: TCM/python_compressibility/normal_h2/cp_normal_h2_ideal.py
def cp_normal_h2_ideal(T):
	from math import exp, pow,log
	ni=2.5   
	ti=0.0
        vi=[1.616,-0.4117,-0.792,0.758,1.217] 
        ui=[531,751,1989,2484,6859]
	term1=0
	term2=0

	#for i in range(0,len(ti)):
	term1=ni*pow(T,ti)
	for i in range(0,len(ui)):
		term2=term2+vi[i]*pow((ui[i]/T),2)*(exp(-ui[i]/T))/pow(1-exp(-ui[i]/T),2)
        return term1+term2

File: TCM/python_compressibility/matlab/cp_normal_h2_ideal.m
function out=cp_normal_h2_ideal(T)
    R=8.314472/2.01594;
	ni=2.5 ;  
	ti=0.0;
    vi=[1.616,-0.4117,-0.792,0.758,1.217] ;
    ui=[531,751,1989,2484,6859];
	term1=0;
	term2=0;
	
	term1=ni*T.^ti;
	for i=1:length(ui)
		term2=term2+vi(i).*((ui(i)./T).^2).*(exp(-ui(i)./T))./((1-exp(-ui(i)./T)).^2);     
    end
    out=R.*(term1+term2)./T;